openedx/core/djangoapps/password_policy/hibp.py

```diff
@@ -0,0 +1,60 @@
+"""
+Wrapper to use pwnedpassword Service
+"""
+
+
+import logging
+
+import requests
+from django.conf import settings
+from requests.exceptions import ReadTimeout
+from rest_framework.status import HTTP_408_REQUEST_TIMEOUT
+
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+
+API_URL = configuration_helpers.get_value('PWNED_PASSWORD_API_URL', settings.PWNED_PASSWORD_API_URL)
```

Comment:
Instead of a global variable, we can move this `PwnedPasswordsAPI` class variable.